app/frontend/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": ["next/core-web-vitals","prettier"]
```

Comment:
**DO NOT REMOVE THE FORMATTER PLEASE!!**

To uphold code quality, readability, and ease of contribution, prettier is super essential. Please add the package and configs back.